### ISSUE TITLE
Display weather by days not by hours

### ISSUE THREAD

**Issue by rikishi0071** [martykan/forecastie#568]
Hi. This app is one of the best FOSS weather apps. But I wonder if it's possible to display weather forecast just by days? I don't want to view weather for every 3 hours, I just want to open the app and show weather day by day. Is it possible to do or OWM API allows to display weather only by every 3 hours?

**Comment by fAntel** [comment]
@rikishi0071 What kind of information you think is better: ranges like "temperature: -5 - -1°" or average like "temperature: -2°"? I was going to implement this kind of information as one of notifications type inspired by one in Weather Notification app. But I'm busy now, and it'll take a while :(

**Comment by rikishi0071** [comment]
Average is better. I liked the style of this app, but when I open it, I can see only weather by every 3 hours. The simplicity is our best friend so I like to see something like: current weather (blue screen on top of the app) and day by day average temperature for the next 5 days or a week (as a list of days on the bottom)

**Comment by robinpaulson** [comment]
I agree, having a daily/three hour option for forecasts would be good

**Comment by robinpaulson** [comment]
Perhaps daily could be the default?

**Comment by rikishi0071** [comment]
Yes. I think daily should be the default.
